src/main/java/org/steam/HomePage.java

```diff
@@ -0,0 +1,50 @@
+package org.steam;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.PageFactory;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import settings.Utilities;
+
+public class HomePage {
+
+    private final WebDriver driver;
+
+    @FindBy(id = "store_nav_search_term")
+    private WebElement searchBar;
+
+    @FindBy(css = "#store_search_link > img")
+    private WebElement searchIcon;
+
+    public HomePage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    public void searchAGame(String gameName) {
```

Comment:
nit : SearchGame gives the save message and less trouble reading than "SearchAGame"